db-server/index.js

```diff
@@ -266,6 +267,13 @@ function createServer(db) {
       res.send(result.map(bufferize.unbuffer))
     }
     else {
+
+      // When performing a `HEAD` request, the content type is not
+      // set, manually set to application/json
+      if (req.method === 'HEAD') {
```

Comment:
Not sure if this is a bug or expected for `HEAD` requests. Only the [/emailRecord/:id](https://github.com/mozilla/fxa-auth-db-mysql/blob/536d413e441b08ffb33ae2073c906f339744d0fa/db-server/index.js#L157) uses `HEAD`. I could modify the test case or add this clause for backwards compat.